### ISSUE TITLE
[DOCS] Prototype page to demo better Superfluid data structure

### ISSUE THREAD

**Issue by vmichalik** [superfluid-finance/docs-v2#35]
## Overview
**What?**
Build a simple demo to visualise a data structure to inform design changes that move us from
Account → SuperToken → Streams
To
Account → SuperTokens → Account-to-Account → All Money Movement Events

**How?**

**Why?**

**Deadline:**

**UI/UX Design:**
N/A

## Acceptance criteria
 - [ ] Satisfies entire acceptance criteria
 - [ ] Consider communication & visibility
 - [ ] Code review is approved by at least 2 reviewers
 - [ ] Tested by Elvijs/QA & discovered bugs resolved
 - [ ] Documentation updated
 - [ ] Assignee/opened by merges PR
 - [ ] Automated E2E tests for happy path
 - [ ] Code is merged
 - [ ] Observability of critical user path
 - [ ] Steps to access code change as a user

## Links/Resources
 * ...

## Readiness criteria
- [ ] Descriptive title
- [ ] What?
- [ ] Why?
- [ ] How?
- [ ] Acceptance criteria complying with Definition of Done
- [ ] Estimate/Story points
- [ ] Deadline: If not a part of an epic
- [ ] Priority: Reflected as order on the board, walking the board from the top and right
- [ ] Assignee
- [ ] Labels
- [ ] Link to epic: If belonging to an epic
- [ ] A task is always a separate ticket/issue
- [ ] UI/UX Design (optional)
- [ ] Blockers/Prerequisites (optional): Use Zenhub blocked by/blocking feature

**Comment by youssefea** [comment]
@kasparkallas do you suggest we have a section in the docs to showcase your tool?

**Comment by youssefea** [comment]
Apologies for closing. Made a mistake.

**Comment by vmichalik** [comment]
Ah not sure how this issue ended up in this repo?
